packages/superset-ui-chart-controls/test/utils/timeCompare.test.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { QueryObject, SqlaFormData } from '@superset-ui/core';
+import { timeCompareTransform } from '../../src';
+
+const formData: SqlaFormData = {
+  metrics: ['count(*)'],
+  time_range: '2015 : 2016',
```

Comment:
Is this valid syntax?

Reply:
yes, It is. you can try on the explore like this:

![image](https://user-images.githubusercontent.com/2016594/124857100-4814f980-dfde-11eb-810c-3c3598cd936c.png)